### PR TITLE
Rework how we load thumbnails

### PR DESCRIPTION
**Pull request by hakubo**
https://wikia-inc.atlassian.net/browse/CONCF-76

Some images are transparent and it looks silly.

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM
@hakubo please provide an example for QA.

File: front/styles/module/article/_media.scss [hunk 1]
```diff
@@ -21,6 +21,10 @@
 		}
 		display: inline-block;
 		width: 100%;
+
+		&.loaded {
+			background: none;
```
Comment (rogatty) [inline front/styles/module/article/_media.scss]: Looks like this line doubles https://github.com/Wikia/mercury/pull/563/files#diff-8b7f8ad08089fbd31dce3abdbacc8b1cR8

File: front/styles/module/article/_media.scss [hunk 2]
```diff
@@ -75,7 +71,7 @@ figcaption {
 	}
 
 	img {
-		height: auto;
+		height: 195px;
```
Comment (rogatty) [inline front/styles/module/article/_media.scss]: Where does it come from? Looks like a good candidate for `_wikia-variables.scss`

**Comment by rogatty** [comment]
* Use vignette-js#2.0.1 when https://github.com/Wikia/vignette-js/pull/22 is merged
* Check how it looks on http://gameshub.wikia.com/wiki/SXSW

**Comment by rogatty** [comment]
https://github.com/Wikia/vignette-js/compare/2.0.0...2.0.1 is good to go

**Comment by rogatty** [comment]
LGTM :+1: 
While doing QA remember to check if http://gameshub.wikia.com/wiki/SXSW looks ok.

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**